iphone/Classes/MediaModule.m

```diff
@@ -1820,13 +1820,15 @@ - (void)popoverPresentationControllerDidDismissPopover:(UIPopoverPresentationCon
 
 #pragma mark UIAdaptivePresentationControllerDelegate
 
+#ifdef USE_TI_MEDIAOPENPHOTOGALLERY
```

Comment:
@ewanharris I think guarding the code inside the delegate method is more relevant.
Guard it with `#if defined(USE_TI_MEDIASHOWCAMERA) || defined(USE_TI_MEDIAOPENPHOTOGALLERY) || defined(USE_TI_MEDIASTARTVIDEOEDITING)
`

Reply:
@vijaysingh-axway fixed!